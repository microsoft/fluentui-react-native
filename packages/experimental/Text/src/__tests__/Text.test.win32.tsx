import * as React from 'react';
import { Text } from '..';
import * as renderer from 'react-test-renderer';

it('Text default', () => {
  const tree = renderer.create(<Text>Text default</Text>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Text all props', () => {
  const tree = renderer
    .create(
      <Text disabled variant="bodyStandard">
        All props
      </Text>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('Text all tokens', () => {
  const BoldText = Text.customize({
    fontFamily: 'Wingdings',
    fontWeight: '900',
    fontSize: 20
  });
  const tree = renderer.create(<BoldText>All tokens</BoldText>).toJSON();
  expect(tree).toMatchSnapshot();
});
