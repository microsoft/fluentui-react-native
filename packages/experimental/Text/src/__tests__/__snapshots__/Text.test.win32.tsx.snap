// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component tests Text all props 1`] = `
<Text
  style={
    Object {
      "color": "#a19f9d",
      "fontFamily": "Segoe UI",
      "fontSize": 14,
      "fontWeight": "400",
      "margin": 0,
    }
  }
>
  All props
</Text>
`;

exports[`Text component tests Text all tokens 1`] = `
<Text
  style={
    Object {
      "color": "#323130",
      "fontFamily": "Wingdings",
      "fontSize": 20,
      "fontWeight": "900",
      "margin": 0,
    }
  }
>
  All tokens
</Text>
`;

exports[`Text component tests Text default 1`] = `
<Text
  style={
    Object {
      "color": "#323130",
      "fontFamily": "Segoe UI",
      "fontSize": 12,
      "fontWeight": "400",
      "margin": 0,
    }
  }
>
  Text default
</Text>
`;
